qtc_packaging/debian_fremantle/control

```diff
@@ -7,11 +7,11 @@ Build-Depends: debhelper (>= 5), libqt4-dev, libx11-dev
 Standards-Version: 3.7.3
 Homepage: http://maemo.org/packages/view/xbmcremote/
 
-Package: xbmcremote
+Package: kodimote
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libqjson, libqtm-12-contacts
```

Comment:
This one seems dangerous to remove too. Are you running into problem if you keep that line? If I remember correctly, you need to install libqjson manually to get the packaging working. Let me know if you have troubles with that, I can help.